docs/en/guides/pipeline-integrations/administration.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: default
+title: Administration
+pagename: toolkit-administration
+lang: en
+---
+
+# Administration
```

Comment:
Should this page have some content? I know it just forms part of the hierarchy, do we feel OK with it being blank when someone clicks on it?

Reply:
added a short description to this page, as well as Development and Administration

Reply:
This is actually a reply to your overarching parent comment; can't seem to reply directly to it:

 - all the dead links have been fixed.
 - fixed the broken images in webhooks.md
 - underscores in image files replaced with hyphens, references to the image files adjusted

Reply:
Great! I've approved it.